packages/client/src/v2-events/features/events/registered-fields/Address.tsx

```diff
@@ -70,7 +75,41 @@ function addDefaultValue<T extends FieldConfigWithoutAddress>(
  * - Address details fields are only shown when district is selected (it being the last admin structure field).
  */
 function AddressInput(props: Props) {
-  const { onChange, defaultValue, value = {}, ...otherProps } = props
+  const {
+    onChange,
+    defaultValue: initialDefaultValue,
+    value = {},
+    ...otherProps
+  } = props
+
+  const userDetails = useSelector(getUserDetails)
+  const locations = useSelector(getLocations)
+
+  const userPrimaryOfficeLocation = {
+    country: 'FAR',
+    province: '',
+    district: '',
+    urbanOrRural: AddressType.URBAN
+  }
```

Comment:
are configurable locations not being considered for the initial release?

Reply:
If you mean configuring the administrative structure then, yes the address fields in the form can be made into what ever structure. It jus means you might not be able to use the pre-built `ADDRESS` component. We've wanted to keep it fairly simple Farajaland like setup for now. 

For future reference, we can easily add more `configuration`s for this component so that the user can for instance define what the fields inbetween country and street address are